example/scripts/postinstall.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
this patches the example projects dependencies so they work with the new react native version and kotlin version - temporary to unblock people, will need to clean up later.